helm-charts/azure-api-management-gateway/values.yaml

```
@@ -14,6 +14,8 @@ nameOverride: ""
 fullnameOverride: ""
 
 gateway:
+  deployment:
+    terminationGracePeriodSeconds: 
```

Let's be consistent as per your point :)
```suggestion
    terminationGracePeriodSeconds: 60
```
Sorry for the pain